libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/maneuver/ManeuverIconDrawer.kt

```diff
@@ -2,12 +2,25 @@ package com.mapbox.navigation.trip.notification.maneuver
 
 import android.graphics.Canvas
 import android.graphics.PointF
+import androidx.annotation.ColorInt
 
+/**
+ * Maneuver icon drawer
+ */
 interface ManeuverIconDrawer {
+    /**
+     * Draw Maneuver icon
```

Comment:
```suggestion
     * Draws the maneuver icon
```

Explain what the manuever icon is?